README.md

```diff
@@ -1,288 +1,133 @@
-  </ul>
-</details>
-
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
+    React.js
+    React , Reduc Toolkit
+    Javascript(ES6)
+    Webpack
+    Node.js
+    jest
+    Html/CSS
 
-> Describe between 1-3 key features of the application.
+## Key Features <a name="key-features"></a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **User can see total continent's countries**
+  **User can view every countinent countries detail's e.g population , language, timezones, region, area, native name**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+-[Live Version](https://world-kx2q.onrender.com/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
+- Create a local directory that you want to clone the repository.
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+- Open the command prompt in the created directory.
 
-### Install
+- On the terminal run this command git clone ``
 
-Install this project with:
+- Go to the repository folder using command prompt cd `World-info-React-Capstone`
 
-<!--
-Example command:
+- Install the dev dependencies for linters run npm install.
+- setup Webpack
 
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!--
-Example command:
+### Prerequisites
 
-```sh
-  rails server
-```
---->
+In order to run this project you need:
 
-### Run tests
+-Any broswer
 
-To run tests, run the following command:
+- knowledge of ES6 modules
+- familiarity with basic git commands
+- working knowledge of webpack
 
-<!--
-Example command:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+### Setup
 
-### Deployment
+Clone this repository to your desired folder:
 
-You can deploy this project using:
+- Open the command prompt in the created directory.
 
-<!--
-Example:
+- On the terminal run this command git clone https://github.com/SabaAhmad404/World-info-React-Capstone
 
-```sh
+- Go to the repository folder using command prompt cd World-info-React-Capstone
 
-```
- -->
+- Install the dev dependencies for linters run npm install.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
+<!-- Author -->
```

Comment:
Mandatory

you should record a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
The video should contain a demo of the project, highlighting all the project features.
your video should contain a highlight about a piece of the code that you find interesting or is proud of making.